tests/phpunit/src/Commands/DocsCommandTest.php

```diff
@@ -0,0 +1,203 @@
+<?php
+
+namespace Acquia\Cli\Tests\Commands;
+
+use Acquia\Cli\Command\DocsCommand;
+use Acquia\Cli\Tests\CommandTestBase;
+use Symfony\Component\Console\Command\Command;
+
+/**
+ * Class DocsCommandTest.
+ *
+ * @property \Acquia\Cli\Command\DocsCommandTest $command
+ * @package Acquia\Cli\Tests\Commands
+ */
+class DocsCommandTest extends CommandTestBase {
+
+  /**
+   * {@inheritdoc}
+   */
+  protected function createCommand(): Command {
+    return $this->injectCommand(DocsCommand::class);
+  }
+
+  /**
+   * Tests the 'docs' command for ACLI.
+   *
+   * @throws \Exception
+   */
+  public function testAcliDocCommand(): void {
+    $this->docCommandValidate([0], '  [0 ] Acquia CLI');
```

Comment:
Need to switch this to use php unit data providers.

Reply:
PR updated with data provider for the test.